packages/map-template/src/components/MapTemplate/MapTemplate.scss

```diff
@@ -47,6 +47,44 @@
     .mi-floor-selector__list {
         box-sizing: unset;
     }
+
+    .gmnoprint {
```

Comment:
Can you elaborate in a code comment why we are doing this?

Reply:
Done ✅ . I will show it to UX department soon for the final approval

Reply:
I don't see it 😄 

Also, can these map-specific styles be placed in a GoogleMapsMap.scss file and a MapboxMap.scss file in their respective component folders?